datacenter/install/aws.md

```diff
@@ -276,6 +184,116 @@ provides multiple advantages to easily deploy and access your application.
       ehazlett/docker-demo:dcus
     ```
 
+
```

Comment:
Note, this isn't new content, I just moved the "How it Works" to be below the setup instructions